src/components/Button.svelte

```diff
@@ -11,6 +11,7 @@
 </script>
 <style>
     .button {
+        margin: 5px;
```

Comment:
Bij reusable components is het makkelijker om `margin` lokaal te zetten als die niet overal nodig is (zoals in de header)